src/components/Event/EventBox.tsx

```diff
@@ -0,0 +1,115 @@
+export interface EventBoxProps {
+  EventName: string;
+  Enroll: boolean;
+  CurrentEnrollment: number;
+  MaxEnrollment: number;
+  ThaiFaculty: string;
+  EngFaculty: string;
+  Date: string;
+  Time: string;
+  Location: string;
+}
+const EnrollBox: React.FC<{
+  CurrentEnrollment: number;
+  MaxEnrollment: number;
+}> = ({ CurrentEnrollment, MaxEnrollment }) => (
+  <div className="flex flex-row gap-2 text-white">
+    <div className="rounded-full bg-pink-500 px-2 py-1 ">
+      <p className="text-xs">ต้องลงทะเบียน/ Limited seats</p>
+    </div>
+    <div className="rounded-full border-2 border-white px-2 py-1">
+      <p className="text-xs">
+        {CurrentEnrollment} / {MaxEnrollment}
+      </p>
+    </div>
+  </div>
+);
+
+const EventBox = (props: EventBoxProps) => {
+  return (
+    <div className="box-border w-72 rounded-lg border-2 border-white bg-gradient-to-br from-gray-100 to-pink-400 shadow-md ring-offset-2 ring-offset-white">
+      <a href="\">
+        <div className="border-1 rounded-md border-black bg-gradient-to-r from-[#393570]  to-[#CA4072] px-4 py-2">
+          <p className="font-ibm text-xl font-semibold uppercase text-white">
+            {props.EventName}
+          </p>
+          {props.Enroll && (
+            <EnrollBox
+              CurrentEnrollment={props.CurrentEnrollment}
+              MaxEnrollment={props.MaxEnrollment}
+            />
+          )}
+        </div>
+        <div className="flex flex-col px-4 py-2 text-start text-white">
+          <p>{props.ThaiFaculty}</p>
+          <p className="text-sm">{props.EngFaculty}</p>
+          <div className="flex flex-col gap-2">
+            <div className="flex gap-2">
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M19 5H18V3C18 2.44772 17.5523 2 17 2C16.4477 2 16 2.44772 16 3V5H8V3C8 2.44772 7.55228 2 7 2C6.44772 2 6 2.44772 6 3V5H5C3.34315 5 2 6.34315 2 8V9H22V8C22 6.34315 20.6569 5 19 5Z"
+                  fill="#FDFFFC"
+                />
+                <path
+                  d="M2 19C2 20.6569 3.34315 22 5 22H19C20.6569 22 22 20.6569 22 19V11H2V19Z"
+                  fill="#FDFFFC"
+                />
+              </svg>
+              {props.Date}
```

Comment:
same to [inconsistency issue](https://github.com/isd-sgcu/oph66-frontend/pull/23/files#diff-38ae16de040485cdeeb1cd128e0e38036c457f4c89d37770ec12694ba9b82553R28)